src/plugins/wb-disable/disable.js

```diff
@@ -30,15 +30,29 @@ var componentName = "wb-disable",
 			$html = wb.html,
 			i18n = wb.i18n,
 			pageUrl = wb.pageUrlParts,
-			li, param;
+			li, param,
+			i18nTemporary = {
```

Comment:
What file? I though you said that the i18n file was an spreadsheet on google drive with edit restricted to a few members.

Is that documented somewhere?